src/platform/site-wide/user-nav/tests/containers/Main.unit.spec.jsx

```diff
@@ -87,6 +88,27 @@ describe('<Main>', () => {
       wrapper.unmount();
     });
 
+    it('should automatically initiate a session log in if a SSOe-flagged user has an active SSOe session but no vets-website session', () => {
+      localStorage.setItem('hasSessionSSO', true);
+      props.useSSOe = true;
```

Comment:
To avoid potentially affecting other tests, rather than set it directly on the `props` object, you can do:

```
const wrapper = shallow(<Main {...props} useSSOe />);
```